content/en/tracing/trace_collection/library_injection_local.md

```diff
@@ -206,6 +206,19 @@ If the application pod fails to start, run `kubectl logs <my-pod> --all-containe
 - **Solution**: Add the `-musl` suffix to the dotnet library version.
 
 
+#### Python installation issues
+
+##### Incompatible Python version
+
+The library injection mechanism for Python only supports injecting the Python library in Python versions 3.7 and above.
+
+
+##### `user-installed ddtrace found, aborting`
+
+- **Problem**: The `ddtrace` library is already installed on the system so the injection logic aborts injecting the library to avoid introducing a breaking change in the application.
+- **Solution**: Remove the installation of `ddtrace` if library injection is desired. Else, use the installed library ([documentation here][26]) instead of library injection.
```

Comment:
```suggestion
- **Solution**: Remove the installation of `ddtrace` if library injection is desired. Otherwise, use the installed library ([see documentation][26]) instead of library injection.
```